### DISCUSSION TITLE
Allow unit tests to be ordered according to your preference

### DISCUSSION THREAD

**Discussion by andressalro** [Kong/insomnia#3965] [Help]
Currently I am using the insomnia test section, I need to mark a launch order of each unit test, then, what I did in the insomnia application is to create first the one that I need to run first, create the second one that I need to run second, and so on. But, it turns out that when I reload the insomnia document the next day, the unit tests get messed up which affects their correct execution (since some depend on others, for example, for unit test number two to work, unit test number one must be executed first), this also happens to me in Inso CLI, causing a mess in the unit tests. Because of this I want to ask if there is a way to specify the order of the unit tests.

**Example:**

I am already in the Insomnia application and I start building my unit tests, I set up this order:
1. First unit test.
2. Second unit test.
3. Third unit test.

The next day, I reopen my document in the Insomnia application and when I review the unit tests I built yesterday, they now come out in a different order:

2.Second unit test.
1.First unit test.
3.Third unit test.

**The order becomes important when having dependent requests and therefore unit tests that depend on each other, in the case of the example, this disorder could trigger a timeout when running my tests, since calling request number two would be implicitly calling request number one, and then calling again request number one associated to unit test number one.**

Thank you very much for your attention

**Comment by dimitropoulos** [comment]
> when I reload the insomnia document the next day, the unit tests get messed up

I agree that it would be nice to reorder.  However, the order _changing_ sounds like a (separate?) bug.

I tried to reproduce this but was unsuccessful.  Any idea how to reproduce the order of the tests changing?
